sub/source/docs/pilot-control.rst

```diff
@@ -0,0 +1,55 @@
+.. _pilot-control:
+
+=============
+Pilot Control
+=============
+
+Control of the vehicle can be accomplished in three ways:
+
+- Via Ground Control Stations using:
+   - GCS attached Joysticks, or command lines, to send MAVLink `RC override <https://mavlink.io/en/messages/common.html#RC_CHANNELS_OVERRIDE>`__ commands that act as RC inputs (even if no RC receiver is attached)
+   - MAVLink ` MANUAL CONTROL <https://mavlink.io/en/messages/common.html#MANUAL_CONTROL>`__ messages which includes axis commands and button states.
+
+- RC control (not enabled by default)
+
+Pilot Control Inputs
+====================
+If RC is not enabled, pilot inputs are via the GCS joystick, controlling the vehicle axes, and body-frame movements laterally,forward/back, and up/down in depth. Additionally, lights and camera can be controlled from the GCS.
+
+If RC is enabled, then the pitch/roll/yaw/throttle/lateral/forward inputs are sent from the RC transmitter and mapped from RC channels to these inputs via the ``RCMAP_`` parameters.
```

Comment:
`RCMAP` params are [noted as disabled](https://github.com/ArduPilot/ardupilot/blame/407b2707d7618bb0c807385e3fbd9ae98d1cf513/ArduSub/Parameters.h#L235) in the Sub firmware, although I'm unsure whether that's still the case given your recent PR to add RC support.

They're also included in [this blacklist](https://github.com/ArduPilot/ardupilot/blob/407b2707d7618bb0c807385e3fbd9ae98d1cf513/Tools/autotest/param_metadata/mdemit.py#L11), but that's for the markdown version of the parameters docs, and may just be outdated.

If these _do_ work now then [the MAVLink Overrides docs](https://github.com/ArduPilot/ardupilot_wiki/blob/master/sub/source/docs/sub-hardware.rst?plain=1#L23) should be updated to mention/reflect that the motion axis channels can be remapped.

Reply:
will update once my RCMAP PR is merged